pkgs/tools/networking/openconnect_pa/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, pkgconfig, vpnc, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, autoreconfHook } :
+
+assert (openssl != null) == (gnutls == null);
+
+let
+  version = "unstable-2018-10-08";
+  name = "openconnect_pa-${version}";
+in stdenv.mkDerivation {
+  inherit name;
+
+  src = fetchFromGitHub {
+    owner = "dlenski";
+    repo = "openconnect";
+    rev = "e5fe063a087385c5b157ad7a9a3fa874181f6e3b";
+    sha256 = "0ywacqs3nncr2gpjjcz2yc9c6v4ifjssh0vb07h0qff06whqhdax"; 
+  };
+
+
```

Comment:
Looking at the other `openconnect` derivation I think it was a good idea to have

```
outputs = [ "out" "dev" ];
```

To lighten it a bit.